swagger-templates/src/main/java/com/okta/swagger/codegen/OktaJavaClientImplCodegen.java

```diff
@@ -150,7 +150,14 @@ private String buildConstructorTypeExtra(CodegenProperty property) {
     @Override
     public CodegenModel fromModel(String name, Schema schema, Map<String, Schema> allSchemas) {
         CodegenModel codegenModel = super.fromModel(name, schema, allSchemas);
-        codegenModel.imports.add(toModelName(codegenModel.classname)); // The 'Default' gets added in the template
+
+        if (codegenModel.classname.equals("CAPTCHAInstance")) {
+            addToModelTagMap("CAPTCHAInstanceLink", "c.ap.tc.ha");
+        } else if (codegenModel.classname.equals("HrefObject")) {
+            addToModelTagMap("HrefObjectHints", "c.ap.tc.ha");
+        } else {
```

Comment:
what does this do specifically?

Reply:
These classes `CAPTCHAInstanceLink` and `HrefObjectHints` needed to be specifically tagged into `com.okta.sdk.resource.c.ap.tc.ha` package this way. Otherwise, they land in `com.okta.sdk.resource.common` package.

Reply:
This code is removed and hence this comment is no longer applicable.